SPECS/glib/glib.spec

```diff
@@ -23,7 +23,7 @@ BuildRequires:  which
 BuildRequires:  python3-pygments
 Requires:       libffi
 Requires:       libselinux
-Requires:       pcre2-devel
```

Comment:
I think this change is the only critical one, since it adds a runtime requirement on a devel package from a non-devel package

Reply:
Talked in DM, removed it from runtime